### DISCUSSION TITLE
특정 장소 / 회의실 또는 원하는 사람 근처로 텔레포트 (빠른 이동)

### DISCUSSION THREAD

**Discussion by joorijung** [metamong-friends/metamong-friends.github.io#47] [제안하기]
왼쪽 장소 / 사람 목록 패널에서 장소나 사람을 클릭해서 해당 위치로 바로 이동할 수 있으면 좋을 것 같아요. 

미팅하고 싶은 사람이 어디있는지 (목록에는 있어도 위치가 어디있는지) 바로 찾기가 어렵고, 또 해당 위치로 직접 이동하기가 번거로운 것 같아서요! 게더에서도 해당 회의실 더블클릭하면 자동으로 이동해주는 기능을 자주 썼어서 말씀드려봅니다. ㅎㅎ

**Comment by ACT-TJ** [comment]
안녕하세요 !

좋은 제안 너무 감사합니다.

저희는 현재 원하는 유저를 클릭하거나 목록에서 유저를 클릭하면 Poke 하여 Poke 받은 유저가 텔레포트를 할 수 있습니다.

제안주신 Flow 와는 반대가 되는 상황인데요. 두 가지 다 쓰임이 있을 듯 하고, 저희가 우선순위로 잡았던 Flow 는 
원하는 상대를 내가 있는 회의실이나 내 곁으로 불러드리는 부분이라고 봐주시면 될 것 같습니다.
혹~시  해당 기능을 아직 사용하지 않으셨다면 활용해 보시면 좋을 것 같습니다! 

거꾸로 내가 원하는 위치로 이동하는 기능도 제안 건으로 관리하여 검토해 보도록 하겠습니다.

감사합니다!!

**Comment by shlee303** [comment]
@joorijung 해당 기능 배포 됐습니다! 좋은 의견 항상 감사합니다!!!!! (회의실로 가는것은 아니고, 유저 클릭하여 이동)
